### ISSUE TITLE
Support of directories

### ISSUE THREAD

**Issue by exislow** [tamland/python-tidal#267]
With the official TIDAL app it is possible to create directories structure and place playlists there. This library currently offers no possibility to query the directory structure. 

Does it make sense to implement this feature, so users can reflect their playlist and directory hierarchy from the app with this library?

**Comment by tehkillerbee** [comment]
Yes, IIRC, we already have an issue for that task. I know it is already implemented in tidal2 for kodi
